packages/flutter_tools/gradle/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ gradlePlugin {
         // The "flutterPlugin" name isn't used anywhere.
         create("flutterPlugin") {
             id = "dev.flutter.flutter-gradle-plugin"
-            implementationClass = "FlutterPlugin"
+            implementationClass = "com.flutter.gradle.FlutterPlugin"
```

Comment:
Consider a comment indicating how someone figures this out. For example would a file move cause this to update?